packages/icons/CHANGELOG.md

```diff
@@ -2,23 +2,24 @@
 
 ## Unreleased
 
+### New Features
+
+-   Add new `row` and `copy` icons. ([#39690](https://github.com/WordPress/gutenberg/pull/39690))
+-   Add new `listItem` icon. ([#39929](https://github.com/WordPress/gutenberg/pull/39929))
+
 ## 8.1.0 (2022-03-23)
 
 ## 8.0.0 (2022-03-11)
 
 ### Breaking Changes
 
--   Changed `dragHandle` footprint from 18x18 to 24x24 to match other icons.  ([#39342](https://github.com/WordPress/gutenberg/pull/39342))
-
-### New Features
-
-- Add new `row` and `copy` icons. 
```

Comment:
I just moved this to the `unreleased section, as it was added three days ago and it was misplaced.